spec/src/main/asciidoc/microprofile-openapi-spec.asciidoc

```diff
@@ -573,6 +577,48 @@ post:
 
 For more samples please see the https://github.com/eclipse/microprofile-open-api/wiki[MicroProfile Wiki].
 
+==== Jakarta Bean Validation Annotations
+
+In some cases, additional schema restrictions can be inferred from Jakarta Bean Validation annotations and used to enhance the generated OpenAPI document.
+
+Vendors are required to process Jakarta Bean Validation annotations and add the properties listed in the table below to the schema model when:
```

Comment:
```suggestion
Implementors are required to process Jakarta Bean Validation annotations and add the properties listed in the table below to the schema model when:
```

Reply:
I'm happy to change this, but "vendors" matches the language used in the rest of the spec, so I'll also raise an issue to consistently talk about implementers (or maybe implementations would be better)